routes/api/v1/foods.js

```diff
@@ -16,4 +16,24 @@ router.get("/", function (req, res) {
     });
 });
 
+router.delete('/:id', function(req, res, next) {
+  Food.destroy({
+    where: {
+      id: req.params.id
+    }
+  })
+	.then(food => {
+     if (food !== 1)
```

Comment:
Hey Rob, I don't think the if statement is working as intended or I don't understand what it's doing. I played around with your branch and I cannot get inside this block. It seems to just to the catch every time.

Reply:
You are right that it isn't clear.  Apparently food returns 1 if it has been deleted, so this conditional throws an error when something hasn't been deleted.  I will revise it to be cleaner.